code/game/atoms_movable.dm

```diff
@@ -848,6 +848,9 @@
 /atom/movable/proc/do_attack_animation(atom/target, atom/tool)
 	set waitfor = 0
 
+	if(!tool)
```

Comment:
Basically what you're trying to do is make it gracefully not fuck up when there's not a tool.
We want it to fuck up so we can find out why though.